src/server.h

```diff
@@ -91,6 +91,12 @@ typedef long long ustime_t; /* microsecond time type. */
 #include "crc64.h"
 
 /* min/max */
+#ifdef min
+#undef min
+#endif
+#ifdef max
+#undef max
+#endif
```

Comment:
AFAIK `undef` can work even if the variable is not defined, so we don't need the `ifdef` / `endif`
am i wrong?